.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+## Changes
+
+<!--
+  Required.
+
+  Briefly describe the changes introduced by this pull request.
+  Also link relevant issues with the "closes", "fixes" or "resolves" keywords,
+  and add co-authors where appropriate with the "co-authored-by" keyword.
+
+  Example:
+  Implemented the functionality to update the user's name.
+  Closes #42
+  Co-authored-by: @octocat
+-->
+
+## Implementation details
+
+<!--
+  Optional.
+
+  If useful for the code review, describe implementation details and, if
+  necessary, why a certain approach was chosen.
+
+  Example:
+  Changes are introduced to the controller and the database model. UI changes
+  are not in the scope of this PR.
+-->
+
+## How to read this PR
+
+<!--
+  Optional.
+
+  If useful for the code review, add some guidance for how to read this PR,
+  including links to preview builds.
+
+  Example:
+  Please start reviewing the changes to the controller files. Then check the
+  changes to the database files. This way, you will have more context about the
+  changes made to the database model.
+  You can test the changes at https://preview-42.example.com/profile/update
```

Comment:
We have this information directly in the GitHub UI:

<img width="885" alt="Screenshot 2022-01-27 at 10 08 51" src="https://user-images.githubusercontent.com/1007051/151327748-88a203a1-30d5-4bd4-bccd-8f92de5bf91d.png">
<img width="868" alt="Screenshot 2022-01-27 at 10 09 07" src="https://user-images.githubusercontent.com/1007051/151327754-d2e4cd0d-d330-4355-8bef-6551814084dd.png">

Reply:
Yes, but that action displays the index of the preview PR.

Here we have also the opportunity to point to an specific page to quickly direct the code reviewer to the place they need to see without having to first navigate to it by themselves.